lib/src/command/outdated.dart

```diff
@@ -536,10 +570,10 @@ class _PackageDetails implements Comparable<_PackageDetails> {
   Map<String, Object> toJson() {
     return {
       'package': name,
-      'current': current?.describe,
-      'upgradable': upgradable?.describe,
-      'resolvable': resolvable?.describe,
-      'latest': latest?.describe,
+      'current': {'version': current?.describe},
```

Comment:
Can we make a json version of `describe` that includes the override information in a structured way?